src/applications/fry-dea/sass/fry-dea.scss

```diff
@@ -24,4 +24,52 @@
 }
 .review .fry-dea-sponsors-labels_label--secondary {
   display: none;
-}
\ No newline at end of file
+}
+
+.information-border {
+  border-left: .5rem solid $color-primary-alt-dark;
+}
+
+
+// Benefit Selection Page
```

Comment:
For anyone else looking As the author of the changes below, I'm not thrilled about having to do this, but it's a lot simper than having to implement the Input Message component (https://design.va.gov/components/form/input-message) by ourselves at the moment.